### ISSUE TITLE
BND3 does not write format correctly

### ISSUE THREAD

**Issue by WarpZephyr** [ividyon/WitchyBND#22]
The format byte of Binder archives can be written in big endian or little endian bit order.  
This is determined by the BitBigEndian byte, which is a true or false value that describes what order format flags are in.  

BND4, BXF3, and BXF4 pass the BitBigEndian field to Binder.WriteFormat to write the format so that it knows if it needs to reverse it.  
BND3 however, does not pass BitBigEndian. BND3 is passing the BigEndian field:  
https://github.com/ividyon/WitchyBND/blob/main/SoulsFormats/Binder/BND3/BND3.cs#L121  

This has been an issue since the original SoulsFormats repo:  
https://github.com/JKAnderson/SoulsFormats/blob/master/SoulsFormats/Binder/BND3/BND3.cs#L121  

Armored Core 4's regulation.bin repacks incorrectly due to this bug since it has the big endian flag set, but big endian and bit big endian are both false. The rest of the BND3 is written how it should have been originally, except the format byte is written backwards.  

Armored Core Verdict Day's ffx_shader.bnd has big endian set to true, but bit big endian and the big endian flag false
I noticed that even though bit big endian was false, true was passed to the WriteFormat method.  

Basically the BitBigEndian field needs to be passed and not the BigEndian field when writing format for BND3.  
Here are those samples:  
[samples.zip](https://github.com/ividyon/WitchyBND/files/14501501/samples.zip)
